warehouse/templates/email/trusted-publisher-added/body.txt

```diff
@@ -22,8 +22,11 @@ users and can create project releases automatically.
 
 {% trans %}Publisher information{% endtrans %}:
 
-* {% trans %}Publisher name{% endtrans %}: {{ publisher_name }}
-* {% trans %}Publisher specification{% endtrans %}: {{ publisher_spec }}
+* {% trans %}Publisher{% endtrans %}: {{ publisher_name }}
+* {% trans %}Workflow{% endtrans %}: {{ publisher_workflow }}
+* {% trans %}Repository{% endtrans %}: {{ publisher_repository }}
+{% if publisher_environment %}* {% trans %}Environment{% endtrans %}: {{ publisher_environment }}
+{% endif %}
```

Comment:
Might be wrong, but I believe this will remove the leading whitespace (note the `-`):

```suggestion
{%- if publisher_environment %}
* {% trans %}Environment{% endtrans %}: {{ publisher_environment }}
{% endif %}
```